pages/terms.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: page-no-toc
+title: "Terms of Use"
+permalink: terms
+---
+
+# Terms of Use
+
+## URI Persistence Policy
+
+Last modified: 2021-01-12
+
+When information is made available on the Web, it is important for the integrity of the Web, and the society based upon it, that the URIs used to reference information be used well into the future, and that the information persist as identified.
+
+To this end, the Solid Project hosts make the following pledge: that as far as they are able, for resources on the solidproject.org Web site which are declared (see below) to be persistent.
+
+Pledge:
+
+* The hosts will ensure that persistent resources continue to be available throughout the life of the Solid Project;
+* Where a persistent resource is modified, a change history will be archived though the archive will not necessarily be available publicly;
+* Should the Solid Project be disbanded, then any Web site will be granted the right to make a copy (at a different URI) of all public persistent resources so long as they are not modified and are preserved in their entirety and made available free of charge, and provided the same persistence policy is applied to these "historical mirrors." In such event, the original https://solidproject.org web site will be handed over for management to another organization only if that organization pledges to this policy or one considered more persistent.
+
+As of this note, persistent resources include:
+
+* The home page "https://solidproject.org/";
+* Those which start "https://solidproject.org/TR/" immediately followed by four decimal digits.
+
+No representation is made about the persistence policies for any other information on the site.
+
+The intent is to set an example by reducing the failure of links due to clumsy management or inadequate commitment to information persistence, and to provide a stable reference base of information about Solid-related topics as a service to the community.
+
+Tim Berners-Lee
```

Comment:
Question for @timbl whether or not this should be signed by "Solid Project" vs individual?

Reply:
Solid Project as an organization I think

Reply:
Updated https://github.com/solid/solidproject.org/pull/489/commits/ae26d5b1550d8b2f1b2f7ffee8c862c84a63f4f6